packages/test/local-server-tests/src/test/noDeltaStream.spec.ts

```diff
@@ -111,13 +111,11 @@ describe("No Delta Stream", () => {
 
         assert.strictEqual(container.connected, true, "container.connected");
         assert.strictEqual(container.clientId, "storage-only client", "container.clientId");
-        assert.strictEqual(container.readonly, true, "container.readonly");
-        assert.strictEqual(container.readonlyPermissions, true, "container.readonlyPermissions");
+        assert.strictEqual(container.readOnlyInfo.readonly, true, "container.readOnlyInfo.readonly");
         assert.ok(container.readOnlyInfo.readonly, "container.storageOnly");
```

Comment:
this duplicates the updated line above.  should probably also update the comment?  guessing that's not right